name: Supervised Regression, features: us_crime
{'X_train': {'cols': 0.1545893719806763, 'rows': 0.9322742474916388}, 'X_valid': {'cols': 0.1592610288262463, 'rows': 0.959866220735786}, 'X_test': {'cols': 0.15755320164137798, 'rows': 0.9398797595190381}}
dataset sizes
(1196, 126) (299, 126) (499, 126)
dropped dataset sizes
(81, 126) (12, 126) (30, 126)
key: 5583, k: 1/4, dataset: 42730, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 21, 'max_bin': 197, 'max_depth': 28, 'min_data_in_leaf': 27, 'learning_rate': 0.05618684721859023, 'num_iterations': 929, 'objective': 'regression', 'verbose': -1, 'num_class': 1}
Training until validation scores don't improve for 100 rounds
Early stopping, best iteration is:
[67]	valid_0's l2: 0.407158
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/basic.py:179: UserWarning: Converting column-vector to 1d array
  _log_warning('Converting column-vector to 1d array')
Traceback (most recent call last):
  File "/home/jahan/Missingness/train.py", line 143, in <module>
    metrics_df, perc_missing = run(
  File "/home/jahan/Missingness/benchmarkaux/openmlrun.py", line 225, in run
    wandb.log({"gbm_rmse_fold{}".format(count):rmse}, commit=False)
UnboundLocalError: local variable 'rmse' referenced before assignment